Core/source/test/test-utils.ts

```diff
@@ -58,33 +58,34 @@ export const expectData = (_data: Uint8Array, type?: 'armoredMsg' | 'msgBlocks'
     expect(dataStr).to.contain('-----BEGIN PGP MESSAGE-----');
     expect(dataStr).to.contain('-----END PGP MESSAGE-----');
   } else if (type === 'msgBlocks') {
-    // tslint:disable-next-line:no-unsafe-any
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
do we actually need this `disable rule`?

Reply:
Yes, next line has `JSON.parse(block)` with unknown structure which leads to `unsafe-return` error.

Reply:
Ok, please check my below comment